fp.js

```diff
@@ -6,7 +6,7 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-people.forEach(person => {
+people.map(function(person) {
   peoplesAges.push(person.age)
 })
```

Comment:
`map` returns a new collection based on its return statement. So it should look something like this...

```js
const ages = people.map(person => {
  // for each person in people, add their age to the ages array
  return person.age;
});
```